bundles/org.openhab.binding.vesync/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,343 @@
+		<state readOnly="true" pattern="%.0f"/>
+	</channel-type>
+
+	<channel-type id="deviceAirQualityBasicType">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Air Quality</label>
+		<description>System representation of air quality</description>
+		<state readOnly="true" pattern="%.0f"/>
+	</channel-type>
+
+	<channel-type id="airQualityPPM25">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Air Quality PPM 2.5</label>
+		<description>Indicator of current air quality</description>
+		<state readOnly="true" pattern="%.0f"/>
+	</channel-type>
+
+	<channel-type id="deviceAFConfigDisplayForever">
+		<item-type>Switch</item-type>
+		<label>Config: Display Forever</label>
+		<description>Configuration: If the devices display is enabled forever</description>
+	</channel-type>
+
+	<channel-type id="deviceAFConfigAutoPrefType">
+		<item-type>String</item-type>
+		<label>Config: Auto Mode Preference</label>
+		<description>The operating mode when the air purifier is set to auto</description>
+		<state readOnly="true">
+			<options>
+				<option value="default">Auto (Air Quality)</option>
+				<option value="quiet">Quiet (No High Speed)</option>
+				<option value="efficient">Auto (Room Size)</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="deviceAFTimerExpiry">
+		<item-type>DateTime</item-type>
+		<label>Auto Off Timer Expiry</label>
+		<description>The time when the auto off timer will be reached</description>
+		<state readOnly="true" pattern="%1$tF %1$tR"/>
+	</channel-type>
+
+	<channel-type id="deviceAFConfigAutoPrefRoomSizeType">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Config: Room size (foot sq)</label>
+		<description>Room size for efficient auto mode</description>
+		<state readOnly="true" pattern="%.0f"/>
+	</channel-type>
+
+	<channel-type id="deviceAFConfigAutoScheduleCountType">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Config: Number of schedules</label>
+		<description>The current number of schedules configured</description>
+		<state readOnly="true" pattern="%.0f"/>
+	</channel-type>
+
+
+	<channel-type id="deviceWaterLackingType">
+		<item-type>Switch</item-type>
+		<label>Water Low or Empty</label>
+		<description>Indicator if the devices water is low or empty?</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="deviceHighHumidityType">
+		<item-type>Switch</item-type>
+		<label>High Humidity</label>
+		<description>Indicator if the device is measuring high humidity?</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="deviceWaterTankLiftedType">
+		<item-type>Switch</item-type>
+		<label>Water Tank Removed</label>
+		<description>Indicator if the device is reporting the water tank as removed?</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="deviceAutomaticStopReachTargetType">
+		<item-type>Switch</item-type>
+		<label>Stop @ Target</label>
```

Comment:
```suggestion
		<label>Setpoint</label>
```

Reply:
This isn't really a setpoint as I see it. A setpoint in my head is the final target temperature to be reached when say heating or cooling terminates. This is really a "Shutdown at Setpoint" as far as I understand it from the protocol. As its referred to as a "Target Humidity", I went with "Stop @ Target" to keep it aligned and the name short? The setpoint for humidity would be: Target Humidity Level. For now keeping as is, happy to change but then the other name should also be updated as well. (It could be Humidity Setpoint for Target Humidity Level, and Auto Off (maybe) instead of Stop @ Target?  The original names seem to describe the functionality however for the protocol?

Reply:
OK, doesn't sound like a setpoint.